src/vs/platform/layout/browser/layoutService.ts

```diff
@@ -9,6 +9,17 @@ import { createDecorator } from 'vs/platform/instantiation/common/instantiation'
 
 export const ILayoutService = createDecorator<ILayoutService>('layoutService');
 
+export interface ILayoutOffsetInfo {
```

Comment:
Looks like TestLayoutService needs to be updated

Reply:
Oh, good catch. Pushed an update